blockchyp/blockchyp_client.py

```diff
@@ -39,6 +42,10 @@ def ping(self, terminal):
         """
         Tests connectivity with a terminal.
         """
+        route = self.__resolve_terminal_route(terminal)
+        reply = self.__terminal_post(route, "/test", payload={"terminalName": terminal})
+
+        return reply
```

Comment:
You should not return the raw reply here, since this leaks the requests abstraction out to the caller. This is a public method, and callers should not need to read the requests documentation in order to handle reply. Reply should probably be the JSON decoded into a dictionary.